docs/organizations/using-coding-standards.md

```diff
@@ -63,7 +54,42 @@ To create a coding standard for your organization:
     !!! note
         A repository can follow one coding standard at a time. Assigning a coding standard to a repository unassigns any previously assigned coding standard.
 
-    ![Applying the coding standard to repositories](images/coding-standard-apply.png)
+    ![Applying the coding standard to repositories](images/coding-standard-apply.png)<!-- TODO ALA-940 screenshot -->
+
+## Configuring a coding standard with advanced settings {: id="advanced-settings"}
+
+If you need more control when configuring a coding standard, use the advanced settings to more precisely define the tools and code patterns that the coding standard should use.
+
+When creating a new coding standard, you can access the advanced settings from the sensitivity selection page by clicking **Advanced settings**. When editing an existing coding standard, the advanced settings are the default view.
+
+To configure a coding standard using advanced settings:
+
+1.  Select the programming languages that the new coding standard should cover.
+
+    !!! important
+        The coding standard will only control tools that can analyze one or more of the languages selected in this step.
+
+        If a language is omitted in this step, Codacy uses repository-level tool configurations if present or global Codacy defaults otherwise, and may analyze the omitted language. To prevent this:
+
+        1. In this step select all languages, so that the coding standard controls all tools.
+
+        1. In the next step, explicitly disable the tools that you don't want to use.
+
+    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)<!-- TODO ALA-940 screenshot -->
+
+1.  Click **Next: Tools and patterns**.
+
+1.  Configure the tools and patterns of the coding standard:
+
+    -   Toggle the tools to run when Codacy analyzes your code.
+    -   For each enabled tool, configure the code patterns to use.
+
+    !!! tip
+        -   Use the filters to find the relevant tools and code patterns. The recommended configurations are manually curated by Codacy or based on     tool defaults and are marked with the icon ![Recommended icon](images/coding-standard-recommended-icon.png).
+        -   To toggle multiple code patterns at once, click the checkbox of the first pattern and **Shift+click** the checkbox of the last pattern in a range.
+        -   To toggle all the code patterns visible on the list, click the checkbox on the header of the code patterns list. If there are more code patterns to load on the list, you can click the link **Enable/Disable all &lt;N&gt; patterns** to toggle all patterns matching the current filters.
+
+    ![Configuring the tools and patterns for the coding standard](images/coding-standard-configure-tools.png)<1-- TODO ALA-940 screenshot -->
 
 ## Defining a default coding standard {: id="set-default"}
```

Comment:
We could mention ([maybe here](https://github.com/codacy/docs/pull/2074/files#diff-b0d95ad8d71d24f01955815d37aca3b3771a1db675011c7753f4fba05f63655bR101)) the option to set a coding standard as default during when creating the it.

Reply:
Sorry, the link doesn't work after I pushed some updates. Where did you suggest that we mention it?

Reply:
Somewhere near this line:
`To set a coding standard as default:`

(GitHub didn't allow me to add that note on the specific line because it wasn't new content... 😉 )